purefa/README.md

```diff
@@ -0,0 +1,130 @@
+
+1. Create a local user on your FlashArray with the Read-Only role and generate an API token for this user.
+   ![Generate an API Key](https://raw.githubusercontent.com/DataDog/integrations-extras/master/purefa/images/API.png) 
+2. Add the following configuration block to the `purefa.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your purefa performance data. See the sample [purefa.d/conf.yaml][4] for all available configuration options.
+
+Note: The `/array` endpoint is required as an abolute minimum when creating your configuration file.
+
+```yaml
+init_config:
+   timeout: 60
+
+instances:
+
+  - openmetrics_endpoint: http://<exporter_ip_or_fqdn>:9491/metrics/flasharray/array?endpoint=<array_ip_or_fqdn>
+    tags:
+       - env:<env>
+       - fa_array_name:<full_fqdn>
+       - host:<full_fqdn>
+    hostname_label: host
+    hostname_format: <HOSTNAME>.<domain_name>
+    headers:
+       Authorization: Bearer <api_token>
+    min_collection_interval: 120
+    empty_default_hostname: true
+    max_returned_metrics: 100000
+
+  - openmetrics_endpoint: http://<exporter_ip_or_fqdn>:9491/metrics/flasharray/volumes?endpoint=<array_ip_or_fqdn>
+    tags:
+       - env:<env>
+       - fa_array_name:<full_fqdn>
+    hostname_label: host
+    hostname_format: <HOSTNAME>.<domain_name>
+    headers:
+       Authorization: Bearer <api_token>
+    min_collection_interval: 120
+    empty_default_hostname: true
+    max_returned_metrics: 100000
+
+  - openmetrics_endpoint: http://<exporter_ip_or_fqdn>:9491/metrics/flasharray/hosts?endpoint=<array_ip_or_fqdn>
+    tags:
+       - env:<env>
+       - fa_array_name:<full_fqdn>
+    hostname_label: host
+    hostname_format: <HOSTNAME>.<domain_name>
+    headers:
+       Authorization: Bearer <api_token>
+    min_collection_interval: 120
+    empty_default_hostname: true
+    max_returned_metrics: 100000
+
+  - openmetrics_endpoint: http://<exporter_ip_or_fqdn>:9491/metrics/flasharray/pods?endpoint=<array_ip_or_fqdn>
+    tags:
+       - env:<env>
+       - fa_array_name:<full_fqdn>
+       - host:<full_fqdn>
+    hostname_label: host
+    hostname_format: <HOSTNAME>.<domain_name>
+    headers:
+       Authorization: Bearer <api_token>
+    min_collection_interval: 120
+    empty_default_hostname: true
+    max_returned_metrics: 100000
+```
+2. [Restart the Agent][5].
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `purefa` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this check.
+
+### Events
+
+The PureFA integration does not include any events.
+
+### Service Checks
+
+The PureFA integration does not include any service checks.
+
+## Support
+
+For support or feature requests, contact Pure Storage through the following methods:
+* Email: pure-observability@purestorage.com
+* Slack: [Pure Storage Code// Observability Channel][11].
```

Comment:
I believe this section is not necessary - for integrations on the main Integrations page, Datadog is listed as the support contact automatically.

Reply:
@anawishnoff question here for you...

Reply:
@sdodsley Apologies for the confusion here, I see now that you're using manifest v2 so the structure of the tile is different (a Support tab with your custom content will be included on your tile)- you can safely ignore my initial comment and keep your support info here. Thanks!